enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -286,8 +346,20 @@ def remove_kernel(self, kernel_id):
         super().remove_kernel(kernel_id)
         self.parent.kernel_session_manager.delete_session(kernel_id)
 
+    def list_kernels(self, tenant_id: Optional[str] = UNIVERSAL_TENANT_ID):
+        """Returns a list of kernel_id's of kernels running."""
```

Comment:
incase, `tenant_id ==  UNIVERSAL_TENANT_ID`, should we not return all the kernels irrespective of `tenant_id`  the individual kernels are associated with.
Without this, 
how will I ever be able to list all kernels running on an EG?

Reply:
Hmm - this is a good point.  You're referring to something like an admin app and not a request from a specific gateway client.  

Perhaps we have another UUID constant that only `list_kernels` checks and, when that tenant_id is present, it ignores tenant_id values and lists all kernels?

Reply:
After a sidebar discussion, we will drop the defaulting of tenant_id to UNIVERSAL_TENANT_ID and display all kernels when nothing is specified - as is the behavior today.